libvast/test/span.cpp

```diff
@@ -0,0 +1,39 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include <string>
+
+#include "vast/detail/byte.hpp"
+#include "vast/detail/narrow.hpp"
+#include "vast/detail/span.hpp"
+
+#include "test.hpp"
```

Comment:
`#define SUITE span` is missing.

In general, this test doesn't cover much of the tested class. Only constructors, `operator[]` and `size()`.